ts/webui/src/index.tsx

```diff
@@ -13,24 +13,24 @@ import * as serviceWorker from './serviceWorker';
 const path = getPrefix();
 
 ReactDOM.render(
-    <Router basename={path === undefined ? null : path}>
-        <Suspense
-            fallback={
-                <div className='loading'>
-                    <img title='loading-graph' src={(path || '') + '/loading.gif'} />
-                </div>
-            }
-        >
-            <Route path='/experiment' component={ExperimentManagerIndex} exact />
-            <Switch>
-                <App>
-                    <Route path='/' component={Overview} exact />
-                    <Route path='/oview' component={Overview} />
-                    <Route path='/detail' component={TrialsDetail} />
-                </App>
-            </Switch>
-        </Suspense>
-    </Router>,
+    <Suspense
+        fallback={
+            <div className='loading'>
+                <img title='loading-graph' src={(path || '') + '/loading.gif'} />
+            </div>
+        }
+    >
+        <Router basename={path === undefined ? '' : path}>
```

Comment:
`path ?? ''`

Reply:
What's the difference between `??` and `||`

Reply:
> What's the difference between `??` and `||`

`??` checks `undefined` and `null`, while `||` checks all falsy value (false, zero, empty string, etc).
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Nullish_coalescing_operator

Reply:
Thanks, get it!